pilot/utils/questionary.py

```diff
@@ -28,14 +28,14 @@ def styled_text(project, question, ignore_user_input_count=False, style=None, hi
             print(color_yellow_bold(f'{user_input.user_input}'))
             return user_input.user_input
 
-    if project.ipc_client_instance is None or project.ipc_client_instance.client is None:
+    if project.check_ipc():
```

Comment:
logic is same as before just instead of checking
`if not self.project.check_ipc():`
I swapped it around so that we are checking 
`if self.project.check_ipc():`